09/25/2013 09:41:58.251 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 657, in <module>
    if __name__ =="__main__":
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 66, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 524, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 290, in battle_system
    end_the_game(room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 508, in end_the_game
    sys.exit()
SystemExit

09/25/2013 09:47:30.111 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 09:49:11.402 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 665, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 09:50:01.586 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 665, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 09:50:40.22 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 665, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 09:52:59.578 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 659, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 10:06:02.503 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 659, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 66, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 524, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 290, in battle_system
    end_the_game(room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 507, in end_the_game
    choice = raw_input('>>>')
EOFError: EOF when reading a line

09/25/2013 10:10:44.992 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 659, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 10:12:43.887 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 660, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 65, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 10:12:44.76 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 660, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 29, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

09/25/2013 10:13:36.282 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 660, in <module>
    except:
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 65, in main
    os.system('CLS')
EOFError: EOF when reading a line

09/25/2013 10:13:36.697 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 659, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 10:28:36.885 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 66, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 528, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 290, in battle_system
    end_the_game(room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 512, in end_the_game
    sys.exit()
SystemExit

09/25/2013 10:31:55.560 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 66, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 528, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 290, in battle_system
    end_the_game(room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 512, in end_the_game
    sys.exit()
SystemExit

09/25/2013 10:48:12.949 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 66, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 528, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 278, in battle_system
    sys.exit()
SystemExit

09/25/2013 10:55:59.365 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 66, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 528, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 217, in battle_system
    command = raw_input('What would you like to do>>>')
EOFError: EOF when reading a line

09/25/2013 11:04:46.862 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 66, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 528, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 277, in battle_system
    raw_input('...')
EOFError: EOF when reading a line

09/25/2013 11:08:05.126 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:08:05.444 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:08:05.853 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:09:45.930 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    if __name__ =="__main__":
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:13:12.43 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 665, in <module>
    logger.write_line([exception_string])
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:13:12.183 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 662, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:13:12.319 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 662, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:16:54.659 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 663, in <module>
    try:
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:16:54.812 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 664, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:16:54.948 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 664, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:27:15.675 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 664, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:28:15.478 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 664, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:34:41.396 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 668, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:35:44.457 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 668, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:35:57.340 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 668, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:38:15.477 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 669, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:38:28.36 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 669, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:40:00.525 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 669, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:40:24.728 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 669, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:41:23.860 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 670, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 54, in main
    player_inventory = game_map.player.inventory[0].values
AttributeError: 'list' object has no attribute 'inventory'

09/25/2013 11:42:21.834 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 55, in main
    player_inventory = game_map.player[0].inventory[0].values
AttributeError: 'inventory_q2class' object has no attribute 'values'

09/25/2013 11:42:51.299 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 405, in describe
    if item in player_inventory:
TypeError: coercing to Unicode: need string or buffer, item_q2class found

09/25/2013 11:43:32.914 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 406, in describe
    if item in player_inventory:
TypeError: coercing to Unicode: need string or buffer, item_q2class found

09/25/2013 11:44:30.76 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 406, in describe
    if item in player_inventory:
TypeError: coercing to Unicode: need string or buffer, item_q2class found

09/25/2013 11:44:56.802 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 407, in describe
    if item in player_inventory:
TypeError: coercing to Unicode: need string or buffer, item_q2class found

09/25/2013 11:45:13.609 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 407, in describe
    if item in player_inventory:
TypeError: coercing to Unicode: need string or buffer, item_q2class found

09/25/2013 11:46:47.101 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:46:51.926 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 37, in main
    exit()
  File "C:\Python27\lib\site.py", line 360, in __call__
    raise SystemExit(code)
SystemExit: None

09/25/2013 11:49:56.74 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 65, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:50:03.410 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 65, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:50:23.407 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 65, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:50:52.791 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 65, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:53:03.376 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 599, in run_command
    sys.exit()
SystemExit

09/25/2013 11:53:42.238 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:53:47.696 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 406, in describe
    if item in player_inventory:
TypeError: coercing to Unicode: need string or buffer, item_q2class found

09/25/2013 11:55:55.123 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 70, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 600, in run_command
    sys.exit()
SystemExit

09/25/2013 11:56:28.343 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:56:57.86 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:56:57.221 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:57:06.832 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 55, in main
    player_inventory.append(game_map.player[0].inventory.value)
AttributeError: 'list' object has no attribute 'value'

09/25/2013 11:57:21.160 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 55, in main
    player_inventory.append(game_map.player.inventory[0].value)
AttributeError: 'list' object has no attribute 'inventory'

09/25/2013 11:57:43.220 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:58:01.302 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:59:24.157 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:59:36.535 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 11:59:44.937 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 55, in main
    player_inventory.append(game_map.player[0].inventory[0:].value)
AttributeError: 'list' object has no attribute 'value'

09/25/2013 12:02:28.817 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 12:48:14.544 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 12:48:14.681 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 12:48:37.961 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 12:52:55.874 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 581, in run_command
    player_inventory.children.append(item)
AttributeError: 'list' object has no attribute 'children'

09/25/2013 12:53:24.512 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    try:
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 12:53:25.62 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 30, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

09/25/2013 12:53:34.410 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 581, in run_command
    game_map.player.inventory.children.append(item)
AttributeError: 'list' object has no attribute 'inventory'

09/25/2013 12:54:05.68 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 581, in run_command
    game_map.player[0].inventory[0].children[0].append(item)
IndexError: list index out of range

09/25/2013 12:54:41.563 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 600, in run_command
    sys.exit()
SystemExit

09/25/2013 12:54:45.746 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 55, in main
    player_inventory.append(game_map.player[0].inventory[0].value)
IndexError: list index out of range

09/25/2013 12:57:36.971 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 12:58:06.56 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 36, in main
    exit()
  File "C:\Python27\lib\site.py", line 360, in __call__
    raise SystemExit(code)
SystemExit: None

09/25/2013 12:58:20.717 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 641, in run_command
    print ", ".join(player_inventory)
TypeError: sequence item 0: expected string, NoneType found

09/25/2013 12:58:57.103 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 406, in describe
    if item in player_inventory:
TypeError: argument of type 'NoneType' is not iterable

09/25/2013 12:59:11.845 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 406, in describe
    if item in player_inventory:
TypeError: coercing to Unicode: need string or buffer, item_q2class found

09/25/2013 13:01:45.729 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 55, in main
    player_inventory = game_map.player[0].inventory[0]
IndexError: list index out of range

09/25/2013 13:02:16.52 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 406, in describe
    if item in player_inventory:
TypeError: argument of type 'inventory_q2class' is not iterable

09/25/2013 13:02:23.891 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 36, in main
    exit()
  File "C:\Python27\lib\site.py", line 360, in __call__
    raise SystemExit(code)
SystemExit: None

09/25/2013 13:02:30.21 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 64, in main
    describe(current_room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 406, in describe
    if item in player_inventory:
TypeError: argument of type 'inventory_q2class' is not iterable

09/25/2013 13:03:06.54 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 670, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 577, in run_command
    player_inventory.append(name)
AttributeError: 'inventory_q2class' object has no attribute 'append'

09/25/2013 13:10:07.388 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 670, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 55, in main
    player_inventory = game_map.state.player[0].inventory[0]
AttributeError: 'map_q2class' object has no attribute 'state'

09/25/2013 13:10:26.235 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 670, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 577, in run_command
    player_inventory.append(name)
AttributeError: 'inventory_q2class' object has no attribute 'append'

09/25/2013 13:16:23.673 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 577, in run_command
    player_inventory.append(name)
AttributeError: 'inventory_q2class' object has no attribute 'append'

09/25/2013 13:16:48.233 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 544, in run_command
    if req == "None" or req in player_inventory:
TypeError: argument of type 'inventory_q2class' is not iterable

09/25/2013 13:17:35.398 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 544, in run_command
    if req == "None" or req in player_inventory:
TypeError: argument of type 'inventory_q2class' is not iterable

09/25/2013 13:18:10.739 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 671, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 13:20:31.472 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 580, in run_command
    player_inventory.item.append(item)
AttributeError: 'inventory_q2class' object has no attribute 'item'

09/25/2013 13:21:06.272 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 580, in run_command
    player_inventory.items.append(item)
AttributeError: 'inventory_q2class' object has no attribute 'items'

09/25/2013 13:21:31.766 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 580, in run_command
    player_inventory.items[0].append(item)
AttributeError: 'inventory_q2class' object has no attribute 'items'

09/25/2013 13:21:54.604 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 580, in run_command
    player_inventory.items[0].append(item)
AttributeError: 'inventory_q2class' object has no attribute 'items'

09/25/2013 13:22:41.279 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    if player_inventory.attrs["items"]:
KeyError: 'items'

09/25/2013 13:22:56.205 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 580, in run_command
    player_inventory[0].item[0].append(item)
TypeError: 'inventory_q2class' object does not support indexing

09/25/2013 13:35:32.679 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 675, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory = player_inventory.attrs[items].split()
TypeError: unhashable type: 'dict'

09/25/2013 13:35:54.387 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory[0].item[0].append(item)
TypeError: 'inventory_q2class' object does not support indexing

09/25/2013 13:36:39.961 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory[0].item[0].append(item)
TypeError: 'inventory_q2class' object does not support indexing

09/25/2013 13:37:42.785 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory.item.append(item)
AttributeError: 'inventory_q2class' object has no attribute 'item'

09/25/2013 13:38:13.630 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory.item.append(name)
AttributeError: 'inventory_q2class' object has no attribute 'item'

09/25/2013 13:38:43.509 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 675, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 579, in run_command
    player_inventory.item.append(name)
AttributeError: 'inventory_q2class' object has no attribute 'item'

09/25/2013 13:39:16.195 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 675, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory = game_map.player.inventory[0]
AttributeError: 'list' object has no attribute 'inventory'

09/25/2013 13:41:25.205 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory = game_map.player.inventory[0]
AttributeError: 'list' object has no attribute 'inventory'

09/25/2013 13:41:44.270 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 579, in run_command
    player_inventory.item.append(name)
AttributeError: 'inventory_q2class' object has no attribute 'item'

09/25/2013 13:42:03.61 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 579, in run_command
    player_inventory.item[0].append(name)
AttributeError: 'inventory_q2class' object has no attribute 'item'

09/25/2013 13:42:34.862 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 579, in run_command
    player_inventory.item[0].append(name)
AttributeError: 'unicode' object has no attribute 'item'

09/25/2013 13:44:18.913 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 581, in run_command
    room.item[0].remove(name)
AttributeError: 'item_q2class' object has no attribute 'remove'

09/25/2013 13:45:05.380 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 581, in run_command
    room.item[0].remove(item)
AttributeError: 'item_q2class' object has no attribute 'remove'

09/25/2013 13:46:17.844 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 675, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 582, in run_command
    room.items[0].children.remove(item)
AttributeError: 'building_q2class' object has no attribute 'items'

09/25/2013 13:47:04.862 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 675, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 13:49:18.639 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 640, in run_command
    print ", ".join(player_inventory)
TypeError

09/25/2013 13:50:06.988 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 66, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 13:51:08.932 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 672, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 640, in run_command
    print ", ".join(player_inventory)
TypeError

09/25/2013 13:51:27.50 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 673, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 544, in run_command
    if req == "None" or req in player_inventory:
TypeError: argument of type 'inventory_q2class' is not iterable

09/25/2013 13:52:18.587 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 30, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

09/25/2013 13:52:27.533 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory.append(name)
AttributeError: 'inventory_q2class' object has no attribute 'append'

09/25/2013 14:03:36.603 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 674, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 69, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 578, in run_command
    player_inventory.append(name)
AttributeError: 'inventory_q2class' object has no attribute 'append'

09/25/2013 14:23:36.969 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 70, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 597, in run_command
    player.inventory[0].value = str(player_inventory)
AttributeError: 'player_q2class' object has no attribute 'inventory'

09/25/2013 14:25:58.420 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 70, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 607, in run_command
    sys.exit()
SystemExit

09/25/2013 14:29:59.701 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 682, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 70, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 609, in run_command
    sys.exit()
SystemExit

09/25/2013 14:36:40.163 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    if __name__ =="__main__":
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:36:40.341 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 682, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:36:40.549 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 682, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:39:09.826 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 676, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 183, in intro
    game_map.intro[0].text[0].value.remove(intro_text)
AttributeError: 'unicode' object has no attribute 'remove'

09/25/2013 14:39:38.467 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 676, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 183, in intro
    game_map.intro[0].value.remove(intro_text)
AttributeError: 'NoneType' object has no attribute 'remove'

09/25/2013 14:39:51.522 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 676, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 183, in intro
    game_map.intro[0].text[0].remove(intro_text)
AttributeError: 'text_q2class' object has no attribute 'remove'

09/25/2013 14:43:01.405 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 196, in intro
    game_map.intro.remove(intro_text)
ValueError: list.remove(x): x not in list

09/25/2013 14:44:00.227 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 198, in intro
    game_map.intro.children.remove(item)
AttributeError: 'list' object has no attribute 'children'

09/25/2013 14:44:45.868 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 197, in intro
    game_map.intro.remove(item)
ValueError: list.remove(x): x not in list

09/25/2013 14:45:11.62 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 197, in intro
    game_map.intro.remove(intro_text)
ValueError: list.remove(x): x not in list

09/25/2013 14:45:30.323 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 197, in intro
    game_map.intro.remove(intro_text)
ValueError: list.remove(x): x not in list

09/25/2013 14:45:59.317 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:46:20.439 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 188, in intro
    if game_map.intro[0].text[0]:
IndexError: list index out of range

09/25/2013 14:46:34.639 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 188, in intro
    if game_map.intro.text:
AttributeError: 'list' object has no attribute 'text'

09/25/2013 14:46:44.903 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 61, in main
    intro()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 188, in intro
    if game_map.intro.text[0]:
AttributeError: 'list' object has no attribute 'text'

09/25/2013 14:47:29.726 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 70, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 619, in run_command
    sys.exit()
SystemExit

09/25/2013 14:48:04.371 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 692, in <module>
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:48:05.536 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:48:25.598 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:49:39.924 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:51:22.759 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:52:22.654 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 682, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 14:53:16.166 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:04:12.205 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:04:28.591 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:04:47.841 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 680, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 67, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:06:04.694 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 649, in run_command
    print ", ".join(player_inventory)
TypeError: sequence item 0: expected string, item_q2class found

09/25/2013 15:06:50.963 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:07:05.977 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:07:34.256 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:11:43.649 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 608, in run_command
    sys.exit()
SystemExit

09/25/2013 15:13:40.374 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:14:19.169 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 608, in run_command
    sys.exit()
SystemExit

09/25/2013 15:14:35.141 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:15:55.337 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 608, in run_command
    sys.exit()
SystemExit

09/25/2013 15:16:39.176 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:16:50.794 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:18:05.22 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:18:44.504 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 553, in run_command
    if req == "None" or req in player.item['name']:
TypeError: list indices must be integers, not str

09/25/2013 15:19:17.160 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 553, in run_command
    if req == "None" or req in player.item.attrs['name']:
AttributeError: 'list' object has no attribute 'attrs'

09/25/2013 15:19:58.902 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 553, in run_command
    if req == "None" or req in player.item.attrs['name']:
AttributeError: 'list' object has no attribute 'attrs'

09/25/2013 15:20:29.269 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 553, in run_command
    if req == "None" or req in player.item.name:
AttributeError: 'list' object has no attribute 'name'

09/25/2013 15:39:07.13 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 608, in run_command
    sys.exit()
SystemExit

09/25/2013 15:41:26.915 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 681, in <module>
    try:
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:41:27.395 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 682, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:42:42.878 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 684, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 611, in run_command
    sys.exit()
SystemExit

09/25/2013 15:42:45.389 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 682, in <module>
    if __name__ =="__main__":
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:43:30.471 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 683, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 589, in run_command
    print game_map.player.item.attrs['name']
AttributeError: 'list' object has no attribute 'item'

09/25/2013 15:44:26.256 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 684, in <module>
    except:
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:44:26.390 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 683, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:47:01.141 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 683, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 610, in run_command
    sys.exit()
SystemExit

09/25/2013 15:47:04.65 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 683, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:47:46.901 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 685, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 592, in run_command
    print game_map.player.item.attrs['name']
AttributeError: 'list' object has no attribute 'item'

09/25/2013 15:48:09.137 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 685, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 592, in run_command
    print game_map.player[0].item.attrs['name']
AttributeError: 'list' object has no attribute 'attrs'

09/25/2013 15:49:00.50 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 593, in run_command
    for things in game_map.player.item.attrs['name']:
AttributeError: 'list' object has no attribute 'item'

09/25/2013 15:49:37.970 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 593, in run_command
    for things in game_map.player[0].item.attrs['name']:
AttributeError: 'list' object has no attribute 'attrs'

09/25/2013 15:50:24.318 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 593, in run_command
    for things in game_map.player.item:
AttributeError: 'list' object has no attribute 'item'

09/25/2013 15:52:40.460 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 593, in run_command
    for things in game_map.player.attrs['item']:
AttributeError: 'list' object has no attribute 'attrs'

09/25/2013 15:52:59.228 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 593, in run_command
    for things in game_map.player[0].attrs['item']:
KeyError: 'item'

09/25/2013 15:53:05.405 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 685, in <module>
    if __name__ =="__main__":
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:53:05.540 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:53:05.674 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:53:05.808 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:53:05.941 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 687, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:53:38.486 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 689, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 595, in run_command
    for things in game_map.player[0].attrs['item']:
KeyError: 'item'

09/25/2013 15:54:25.205 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 689, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:54:41.825 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 689, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 37, in main
    exit()
  File "C:\Python27\lib\site.py", line 360, in __call__
    raise SystemExit(code)
SystemExit: None

09/25/2013 15:55:02.740 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 689, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:56:04.44 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 686, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 613, in run_command
    sys.exit()
SystemExit

09/25/2013 15:57:09.852 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 686, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 15:58:15.722 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 686, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:00:11.30 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 685, in <module>
    try:
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:00:11.286 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 686, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:01:31.950 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 686, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:02:04.39 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 686, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 613, in run_command
    sys.exit()
SystemExit

09/25/2013 16:02:37.313 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 686, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:24:44.737 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 688, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 615, in run_command
    sys.exit()
SystemExit

09/25/2013 16:25:08.657 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 688, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:27:42.807 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 670, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:28:11.224 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 670, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/text_of_mana_party_test.py", line 63, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:29:40.776 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 688, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 614, in run_command
    sys.exit()
SystemExit

09/25/2013 16:31:11.995 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 689, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 657, in run_command
    print inv.attrs['name']
AttributeError: 'str' object has no attribute 'attrs'

09/25/2013 16:33:02.842 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 690, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:34:11.446 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:34:40.679 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 547, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 220, in battle_system
    if "rusty sword" in inv:
TypeError: argument of type 'item_q2class' is not iterable

09/25/2013 16:35:35.851 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 31, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

09/25/2013 16:35:48.484 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:46:14.742 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 31, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

09/25/2013 16:48:15.595 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 617, in run_command
    sys.exit()
SystemExit

09/25/2013 16:48:28.905 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:48:37.418 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 68, in main
    command = raw_input(">")
EOFError: EOF when reading a line

09/25/2013 16:52:17.585 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 31, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

09/25/2013 16:55:20.346 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 691, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 547, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 304, in battle_system
    end_the_game(room)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 530, in end_the_game
    sys.exit()
SystemExit

09/25/2013 17:16:20.280 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 690, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 71, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 546, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/inventory play.py", line 291, in battle_system
    sys.exit()
SystemExit

